src/llm_vm/onsite_llm.py

```diff
@@ -172,7 +172,7 @@ def asynctune():
                 timestamp = datetime.now().strftime('%Y-%m-%dT%H-%M-%S')
             else:
                 timestamp = datetime.now().strftime('%Y-%m-%dT%H:%M:%S')
-            new_model = os.path.join(model_path_default,"finetuned_models",self.model_name, timestamp + '_' + self.model_name + ".pt" )
+            new_model = os.path.join(model_path_default,"finetuned_models",self.model_name, timestamp + '_' + self.model_name + ".pt" ) if model_filename is None else os.path.join(model_path_default,"finetuned_models",model_filename)
```

Comment:
so this has the issue that it clobbers the loaded model with the fine tuned name right
?